### PR TITLE
link and screenshot update for Visual Studio 2017

### PR DESCRIPTION
**Pull request by Mikejo5000**
Updated screenshot to VS 2017.
Also, link to Server Hostingh bundle is to a Preview;
For ASP.NET Core templates in VS 2017 to run, you need the more recent version of the SDK.

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@Mikejo5000  looks good. Can you sign the CLA? cc @GuardRex

File: aspnetcore/publishing/iis.md [hunk 1]
```diff
@@ -54,6 +54,9 @@ Proceed through the **Confirmation** step to install the web server role and ser
 
 1. Install the [.NET Core Windows Server Hosting](https://go.microsoft.com/fwlink/?linkid=837808) bundle on the hosting system. The bundle will install the .NET Core Runtime, .NET Core Library, and the [ASP.NET Core Module](xref:fundamentals/servers/aspnet-core-module). The module creates the reverse-proxy between IIS and the Kestrel server. Note: If the system doesn't have an Internet connection, obtain and install the *[Microsoft Visual C++ 2015 Redistributable](https://www.microsoft.com/download/details.aspx?id=53840)* before installing the .NET Core Windows Server Hosting bundle.
 
+    > [!NOTE]
+    > To run ASP.NET templates in Visual Studio 2017, you also need to install the [.NET Core SDK](https://www.microsoft.com/net/download/core).
```
Comment (GuardRex) [inline aspnetcore/publishing/iis.md]: Would "use" be better than "run" here?

**Comment by GuardRex** [comment]
@Mikejo5000 Thank you sooooo much for replacing that image. I :heart: it!

File: aspnetcore/publishing/iis.md (hunk 1 above)
Comment (GuardRex) [inline aspnetcore/publishing/iis.md]: The only other ? I have about this is that the SDK should be installed with VS2017 auto-magically, no?

**Comment by GuardRex** [comment]
The server hosting bundle link: `https://go.microsoft.com/fwlink/?linkid=837808` is the latest we were provided. @shirhatti is that correct?

**Comment by Rick-Anderson** [comment]
@GuardRex  should I merge this and open an issue with your other comments?

**Comment by GuardRex** [comment]
Idk why a separate install of the SDK would be required to use the VS templates when the VS install gives you the SDK. AFAIK that note isn't required.

AFAIK the link we have for the hosting bundle is correct, @shirhatti can confirm.

**Comment by Mikejo5000** [comment]
I experienced the issue with the SDK on both a WS 2012 R2 and a WS 2016 machine; I assume it was because the "Server Hosting" bundle appears to be a preview release.

**Comment by Mikejo5000** [comment]
OK--I rechecked the Installer; I had installed .NET and the ASP.NET workloads, but I didn't notice there was a separate installation for ASP.NET Core workload. So yes, the note appears unnecessary.

**Comment by GuardRex** [comment]
I did *finally* catch up with what you meant ... you were saying that the hosting bundle installer *on a remote machine* doesn't provide the same version of the SDK that you used when compiling the app. I was just a bit confused by your OP when you said "templates in VS 2017 to run" ... I thought you meant locally. I thought you were saying the hosting bundle installer link was old was a separate problem. Ok ... I'm with you now. :+1:

@Rick-Anderson This LGTM. Thanks @Mikejo5000 for fixing that image. It was haunting :ghost: me.

**Comment by shirhatti** [comment]
>The server hosting bundle link: https://go.microsoft.com/fwlink/?linkid=837808 is the latest we were provided. @shirhatti is that correct?

https://go.microsoft.com/fwlink/?linkid=844461

**Comment by GuardRex** [comment]
@Mikejo5000 Would you like to change the server hosting bundle link? If not, no worries ... I'll do a separate PR.

**Comment by Mikejo5000** [comment]
Sure, I can make those changes

**Comment by GuardRex** [comment]
@Mikejo5000 Thanks much!

@Rick-Anderson LGTM :shipit:

**Pull request merged**